javascripts/googleAnalyticsImporter.js

```diff
@@ -0,0 +1,91 @@
+(function () {
```

Comment:
@elabuwa I am unable to see the notification when the import is running
![Uploading Screenshot from 2022-11-17 12-20-41.png…]()

Reply:
HTTP request
URL: `https://localhost.matomo.com/index.php?idSite=311&date=2022-11-17&period=day&module=GoogleAnalyticsImporter&action=pendingImports`
Response: `{"displayPending":false}`

Reply:
HI @AltamashShaikh , can you verify the conditions I mentioned in https://github.com/matomo-org/plugin-GoogleAnalyticsImporter/pull/295#pullrequestreview-1183755856 and confirm please.

Reply:
Verified, it works now as expected